cuda_core/cuda/core/experimental/_strided_copy/_jit.pyx

```diff
@@ -0,0 +1,105 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from libc.stdint cimport intptr_t
+
+import os
+import threading
+
+from cuda.core.experimental._device import Device
+from cuda.core.experimental._program import Program, ProgramOptions
+
+
+_tls = threading.local()
+
+# In multithreaded environment we share the compiled and loaded modules between threads.
+# Each thread has its own cache mapping arch -> kernel_code_str -> Kernel ptr,
+# on a cache miss, we first take a look into the shared cache guarded with _kernel_lock
+# and eventually compile if needed.
+_kernel_lock = threading.Lock()
+_kernel_cache = {}  # arch -> kernel_code_str -> Kernel
+
+
+cpdef str get_strided_copy_include_dir(object logger):
+    """
+    Finds and caches the absolute path for the strided copy includes.
+    """
+    # TODO(ktokarski) Once Program API supports passing includes as strings and names,
+    # read all the headers once and cache them.
+    if hasattr(_tls, "strided_copy_include_dir"):
+        return _tls.strided_copy_include_dir
```

Comment:
```suggestion
    if (strided_copy_include_dir := getattr(_tls, "strided_copy_include_dir", None)) is not None:
        return strided_copy_include_dir
```

This is one attribute access versus two (one inside `hasattr`, and then the attribute access itself).

Feel free to not use the `:=` walrus operator if you don't like it.